ansible/roles/install-microshift/defaults/main.yml

```diff
@@ -55,3 +62,7 @@ du_dirs:
   - /var/lib/containers/storage/
   - /usr/
   - /usr/bin/*
+
+lvm_disk: /dev/xvdb
```

Comment:
This is the device name on AWS cloud, right? However, these are different in other environment, specifically in VMs created by libvirt.